src/firmware/setup-and-install.md

```diff
@@ -8,17 +8,17 @@ Download the [latest Visual Studio Code](https://code.visualstudio.com/download)
 
 ### Download
 
-![img](https://i.imgur.com/jXPXIFz.gif)
+![Downloading Visual Studio Code](https://i.imgur.com/jXPXIFz.gif)
```

Comment:
Should we also upload all gifs to assets too? Just so we don't rely on any outside hosts anymore, since problems happen.